tests/utils/focus/getActiveElement.ts

```diff
@@ -0,0 +1,32 @@
+import '../../_helpers/shadow-input'
+import {setup} from '#testHelpers'
+import {getActiveElement, getActiveElementOrBody} from '#src/utils'
+
+describe('focus tests with not shadow root', () => {
+  test('focus input element', async () => {
+    const {element} = setup('<input />')
+
+    expect(getActiveElement(document)).toBe(element)
```

Comment:
```suggestion
    expect(getActiveElementOrBody(document)).toBe(element)
```

We should test through the util that is probably what should be used in all our APIs when retrieving the focused element.

Reply:
I did change all usages to use `getActiveElementOrBody`. Are users expected to use this? I would love to remove the export for `getActiveElement`, but this would be a breaking change, so holding back on that for now.